base/uk.ac.stfc.isis.ibex.alarm/src/uk/ac/stfc/isis/ibex/alarm/Alarm.java

```diff
@@ -139,12 +140,14 @@ public AlarmCounter getCounter() {
      * connection is closed on a separate thread; give it a chance to close
      * first.
      * 
-     * @return an active MQ connection closer
+     * @return an active MQ connection closer; or null if there is no alarmModel to release.
```

Comment:
We should use an `Optional` rather than returning `null`.

Reply:
Probably not super important for this change so merging anyway. But for future new code please use Optionals :)